### ISSUE TITLE
Some confusion about code

### ISSUE THREAD

**Issue by LeePleased** [andy194673/nlg-sclstm-multiwoz#5]
Hello, I'm a graduate candidate of HIT-SCIR lab. I sincerely thank you for the implementaion of SCLSTM by pytorch, which save us from tensorflow and theano. But i found some serious problems as follows:

1, It seems that the logic of computing slot ERR rate is not correct and alwasys get 100% error rate;
2, It seems that your loss function is only comprised by cross entropy  without penalty for evolution of dt;
3, I'm not sure if template file is reasonable as position is defined in format d-a-s-v:xxxxxx. suck as 
    d-a-s-v:Attraction-Inform-Addr-3, in real word diagloue turn, 3 may not enough.

Great thanks for your contribution about this repository.
Lee 19/03/09
